test.js

```diff
@@ -24,8 +25,8 @@ test('filnamify()', t => {
 	t.is(filenamify('c/n', {replacement: 'con'}), 'cconn');
 });
 
-test('filenamify.path()', t => {
-	t.is(path.basename(filenamify.path(path.join(__dirname, 'foo:bar'))), 'foo!bar');
+test('filenamifyPath()', t => {
+	t.is(path.basename(filenamifyPath(path.join(dirname(import.meta), 'foo:bar'))), 'foo!bar');
```

Comment:
I would prefer just using URL resolution instead of a package just for this.

```js
import {fileURLToPath} from 'node:url';

const foo = fileURLToPath(new URL('foo.js', import.meta.url));
```